test/TEST-01-BASIC/test.sh

```diff
@@ -58,7 +58,7 @@ Type=oneshot
 EOF
 
         setup_testsuite
-    )
+    ) || return 1
```

Comment:
I want to use [the Unofficial Bash Strict Mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/). But it requires many changes.